ui/components/app/assets/nfts/nfts-items/nfts-items.tsx

```diff
@@ -0,0 +1,300 @@
+import React, { useContext, useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useHistory } from 'react-router-dom';
+import { isEqual } from 'lodash';
+import { Hex } from '@metamask/utils';
+import {
+  Display,
+  FlexWrap,
+  BlockSize,
+  FlexDirection,
+} from '../../../../../helpers/constants/design-system';
+import { ENVIRONMENT_TYPE_POPUP } from '../../../../../../shared/constants/app';
+// TODO: Remove restricted import
+// eslint-disable-next-line import/no-restricted-paths
+import { getEnvironmentType } from '../../../../../../app/scripts/lib/util';
+import { getCurrentChainId } from '../../../../../../shared/modules/selectors/networks';
+import {
+  getIpfsGateway,
+  getSelectedInternalAccount,
+  getCurrentNetwork,
+} from '../../../../../selectors';
+import {
+  ASSET_ROUTE,
+  SEND_ROUTE,
+} from '../../../../../helpers/constants/routes';
+import { getAssetImageURL } from '../../../../../helpers/utils/util';
+import { getNftImageAlt } from '../../../../../helpers/utils/nfts';
+import { updateNftDropDownState } from '../../../../../store/actions';
+import { usePrevious } from '../../../../../hooks/usePrevious';
+import { getNftsDropdownState } from '../../../../../ducks/metamask/metamask';
+import { useI18nContext } from '../../../../../hooks/useI18nContext';
+import { Box, Text } from '../../../../component-library';
+import { NftItem } from '../../../../multichain/nft-item';
+import {
+  getSendAnalyticProperties,
+  updateSendAsset,
+} from '../../../../../ducks/send';
+import { AssetType } from '../../../../../../shared/constants/transaction';
+import { MetaMetricsContext } from '../../../../../contexts/metametrics';
+import {
+  MetaMetricsEventCategory,
+  MetaMetricsEventName,
+} from '../../../../../../shared/constants/metametrics';
+import { isEqualCaseInsensitive } from '../../../../../../shared/modules/string-utils';
+import {
+  Collection,
+  NFT,
+} from '../../../../multichain/asset-picker-amount/asset-picker-modal/types';
+import { PreviouslyOwnedCollections } from '../../../../multichain/asset-picker-amount/asset-picker-modal/asset-picker-modal-nft-tab';
+
+type NftsItemsProps = {
+  collections: Record<string, Collection>;
+  previouslyOwnedCollection: PreviouslyOwnedCollections;
+  isModal?: boolean;
+  onCloseModal?: () => void;
+  showTokenId?: boolean;
+  displayPreviouslyOwnedCollection?: boolean;
+};
+
+const width = (isModal: boolean) => {
+  const env = getEnvironmentType() === ENVIRONMENT_TYPE_POPUP;
+
+  if (isModal) {
+    return BlockSize.OneThird;
+  }
+  if (env === Boolean(ENVIRONMENT_TYPE_POPUP)) {
```

Comment:
This is equivalent to: `true === true`
Can we do a small refactor?
```ts
const isPopup = getEnvironmentType() === ENVIRONMENT_TYPE_POPUP
...

if (isPopup) {
 ...
}
```

Reply:
Removed `nft-items` component since it was no longer being used. Going to resolve this.